src/app/modules/page-header/page-header.module.ts

```diff
@@ -11,11 +13,14 @@ import { PageTitleHeaderComponent } from 'app/modules/page-header/page-title-hea
     FlexModule,
     RouterModule,
     TranslateModule,
+    CommonDirectivesModule,
   ],
   declarations: [
+    BreadcrumbComponent,
     PageTitleHeaderComponent,
   ],
   exports: [
+    BreadcrumbComponent,
```

Comment:
do we really need to export `BreadcrumbComponent`?

Reply:
We are using the PageHeaderModule to import into the LayoutModule and EntityModule and in this case we need to migrate the BreadcrumbComponent to PageHeaderModule otherwise it will not be available in the PageTitleHeaderComponen template